### ISSUE TITLE
Bug: Clicking a long alt text in an image get's the app stuck

### ISSUE THREAD

**Issue by DerMolly** [Dimillian/IceCubesApp#951]
## Environment:

 - OS: 
macOS 13.0
 - IceCubesApp version: 
1.5.3 (1466)

## Description

When clicking the alt icon on an image and the alt text is long the app is grayed out and you're not able to interact with the app. Even a normal quit app does not work. Force quitting was the only solution I could find.

The toot where I noticed the problem: https://toot.kif.rocks/@DerMolly/109885048108394251

## Related Issues

 - [x] Search that this bugs don't already exist before creating it.
 

![image](https://user-images.githubusercontent.com/3978662/219963177-139d789c-a051-4ee0-a0df-c563028dee7b.png)
